src/sentry/api/serializers/models/group.py

```diff
@@ -110,6 +110,7 @@ class BaseGroupResponseOptional(TypedDict, total=False):
     userCount: int
     firstSeen: datetime
     lastSeen: datetime
+    lifetime: dict[str, Any]
```

Comment:
From reading this variable, it's hard to figure out its purpose since it's an adjective rather than a noun.

Reply:
hmm - i would prefer not to change this given how it's already being used in quite a few places. while the variable is just lifetime, if you look at the value it has (example: ` lifetime: {count: 10, userCount: 5}`) , i think it makes sense